src/l500/l500-depth.h

```diff
@@ -178,6 +178,12 @@ namespace librealsense
             intrinsics.fy = intrinsic_params.pinhole_cam_model.ipm.focal_length.y;
             intrinsics.ppx = intrinsic_params.pinhole_cam_model.ipm.principal_point.x;
             intrinsics.ppy = intrinsic_params.pinhole_cam_model.ipm.principal_point.y;
+            intrinsics.coeffs[0] = 0;
```

Comment:
@jangernert hi , 
it might not necessary apply here, but in general case the coefficients should be copied from the source rather than assigned.
You can either change it (preferred) or add explanation in code why it can be avoided in the particular case.

Reply:
I honestly didn't check if the distortion struct had any valid values since the image is not distorted.